### PR TITLE
Update data test id attributes

### PR DESCRIPTION
**Pull request by karola312**
I want to merge this change because I updated data-test-id

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://stable.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-5141 Update test attributes names](https://app.clickup.com/t/2549495/SALEOR-5141)

**Review by jwm0 — APPROVED** [review]

File: cypress/elements/giftCard/giftCardDialog.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 export const GIFT_CARD_DIALOG = {
   amountInput: '[name="balanceAmount"]',
   currencySelectButton: '[id="mui-component-select-balanceCurrency"]',
-  currenciesOptions: '[data-test="selectFieldOption"]',
+  currenciesOptions: '[data-test-id*="select-field-option"]',
```
Comment (eCzerniak) [inline cypress/elements/giftCard/giftCardDialog.js:4]: Line 16: 
`cardCodeText: '[data-test-id="card-code"]',`

File: cypress/elements/account/left-menu/left-menu-selectors.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 export const LEFT_MENU_SELECTORS = {
   catalog: "[data-test='menu-item-label'][data-test-id='catalogue']",
+  pages: "[data-test='menu-item-label'][data-test-id='pages']",
   configuration: "[data-test='menu-item-label'][data-test-id='configure']",
```
Comment (eCzerniak) [inline cypress/elements/account/left-menu/left-menu-selectors.js:4]: Line 18: 
`draftOrders: "[data-test='submenu-item-label'][data-test-id='order-drafts']"`

File: cypress/elements/account/login-selectors.js [hunk 3]
```diff
@@ -1,9 +1,8 @@
 export const LOGIN_SELECTORS = {
-  accountSettings: "[data-test=accountSettingsButton]",
+  accountSettings: "[data-test-id=account-settings-button]",
   emailAddressInput: "input[name='email']",
   emailPasswordInput: "input[name='password']",
-  signInButton: "[data-test=submit]",
+  signInButton: "[data-test-id=submit]",
   userMenu: "[data-test=userMenu]",
```
Comment (eCzerniak) [inline cypress/elements/account/login-selectors.js:6]:
```suggestion
  userMenu: "[data-test=user-menu]",
```

File: cypress/elements/orders/orders-selectors.js [hunk 4]
```diff
@@ -1,10 +1,11 @@
...
-  orders: "[data-test='submenu-item-label'][data-test-id='orders']",
+  orders: "[data-test-id='submenu-item-label'][data-test-id='orders']",
   createOrder: "[data-test-id='create-order-button']",
   orderRow: "[data-test-id='order-table-row']",
   salesChannel: "[data-test-id='order-sales-channel']",
   cancelFulfillment: "[data-test-id='cancelFulfillmentButton']",
-  cancelFulfillmentSelectField: "[data-test-id='cancelFulfillmentSelectField']",
+  cancelFulfillmentSelectField:
+    "[data-test-id='cancel-fulfillment-select-field']",
   orderFulfillmentFrame: "[data-test-id='orderFulfillment']",
```
Comment (eCzerniak) [inline cypress/elements/orders/orders-selectors.js:9]:
```suggestion
  orderFulfillmentFrame: "[data-test-id='order-fulfillment']",
```

File: cypress/elements/shared/button-selectors.js [hunk 5]
```diff
@@ -1,16 +1,16 @@
...
-  submit: '[data-test="submit"], [data-test-id="submit"]',
+  submit: '[data-test-id="submit"]',
   confirm: '[data-test="button-bar-confirm"]',
   goBackButton: "[data-test-id='app-header-back-button']",
   checkbox: "[type='checkbox']",
   checked: "[class*='checked']",
-  selectOption: "[data-test*='select-option']",
+  selectOption: "[data-test-id*='select-option']",
   notSelectedOption: ":not([aria-selected])",
   deleteButton: '[data-test="button-bar-delete"]',
```
Comment (eCzerniak) [inline cypress/elements/shared/button-selectors.js:10]:
```suggestion
  deleteButton: '[data-test-id="button-bar-delete"]',
```

File: cypress/elements/shared/button-selectors.js [hunk 6]
```diff
@@ -1,16 +1,16 @@
...
   expandIcon: '[data-test-id="expand-icon"]',
   nextPaginationButton: '[data-test="button-pagination-next"]',
```
Comment (eCzerniak) [inline cypress/elements/shared/button-selectors.js:12]:
```suggestion
  nextPaginationButton: '[data-test-id="button-pagination-next"]',
```

File: cypress/elements/translations/element-translation.js [hunk 7]
```diff
@@ -3,5 +3,5 @@ export const ELEMENT_TRANSLATION = {
   editDescriptionButton: '[data-test-id="edit-description"]',
   editSeoTitleButton: '[data-test-id="edit-seoTitle"]',
```
Comment (eCzerniak) [inline cypress/elements/translations/element-translation.js:4]:
```suggestion
  editSeoTitleButton: '[data-test-id="edit-seo-title"]',
```

File: cypress/elements/translations/element-translation.js [hunk 8]
```diff
@@ -3,5 +3,5 @@ export const ELEMENT_TRANSLATION = {
...
   editSeoDescriptionButton: '[data-test-id="edit-seoDescription"]',
```
Comment (eCzerniak) [inline cypress/elements/translations/element-translation.js:5]:
```suggestion
  editSeoDescriptionButton: '[data-test-id="edit-seo-description"]',
```

**Review by eCzerniak — CHANGES_REQUESTED** [review]

File: cypress/elements/configuration/configuration-selectors.js [hunk 9]
```diff
@@ -1,15 +1,15 @@
...
-  navigation: '[data-test-id="configurationMenuNavigation"]',
-  staffMembers: '[data-test-id="configurationMenuStaff"]',
-  permissionGroups: '[data-test-id="configurationMenuPermissionGroups"]',
-  attributes: '[data-test-id="configurationMenuAttributes"]',
-  productTypes: '[data-test-id="configurationMenuProductTypes"]',
-  plugin: '[data-test-id="configurationPluginsPages"]',
-  warehouse: '[data-test-id="configurationMenuWarehouses"]'
+  channels: "[data-test-id*='channels']",
+  shipping: '[data-test-id*="shipping methods"]',
+  taxes: '[data-test-id*="configurationMenuTaxes"]',
```
[the remark this answers is not in the document]
Comment (karola312) [reply cypress/elements/configuration/configuration-selectors.js]: changed

File: cypress/elements/giftCard/giftCardDialog.js (hunk 1 above)
Comment (karola312) [reply cypress/elements/giftCard/giftCardDialog.js]: I removed file for gift card - I will fix it when merging to 3.1

File: cypress/elements/account/left-menu/left-menu-selectors.js (hunk 2 above)
Comment (karola312) [reply cypress/elements/account/left-menu/left-menu-selectors.js:4]: Can't change data-test='submenu-item-label', but I fixed data-test-id='order drafts'

**Review by eCzerniak — APPROVED** [review]

**Pull request merged**